flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
this is disappointing.

Reply:
See https://github.com/flathub/flathub/pull/3272#discussion_r897950584